DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Operations/Send/RevitRootObjectBuilder.cs

```diff
@@ -8,24 +8,17 @@
 
 namespace Speckle.Connectors.Revit.Operations.Send;
 
-public class RootObjectBuilder : IRootObjectBuilder<ElementId>
+public class RevitRootObjectBuilder : IRootObjectBuilder<ElementId>
 {
   // POC: SendSelection and RevitConversionContextStack should be interfaces, former needs interfaces
-  private readonly ISpeckleConverterToSpeckle _converter;
-  private readonly ToSpeckleConvertedObjectsCache _convertedObjectsCache;
```

Comment:
Was the object cache not being used at all?

Reply:
It wasn't!